[2022-01-25 19:47:06,127] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.check manual__2022-01-25T19:47:04.811403+00:00 [queued]>
[2022-01-25 19:47:06,155] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.check manual__2022-01-25T19:47:04.811403+00:00 [queued]>
[2022-01-25 19:47:06,156] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 19:47:06,157] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-25 19:47:06,157] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 19:47:06,187] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): check> on 2022-01-25 19:47:04.811403+00:00
[2022-01-25 19:47:06,199] {standard_task_runner.py:52} INFO - Started process 4472 to run task
[2022-01-25 19:47:06,217] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'check', 'manual__2022-01-25T19:47:04.811403+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/dag_01.py', '--cfg-path', '/tmp/tmp9h3s30me', '--error-file', '/tmp/tmpxncswfdz']
[2022-01-25 19:47:06,218] {standard_task_runner.py:77} INFO - Job 102: Subtask check
[2022-01-25 19:47:06,365] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.check manual__2022-01-25T19:47:04.811403+00:00 [running]> on host 223810fcea28
[2022-01-25 19:47:06,479] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=check
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T19:47:04.811403+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-25T19:47:04.811403+00:00
[2022-01-25 19:47:06,481] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 19:47:06,482] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo -------------------------- EMPIEZA ------------------------ && pwd']
[2022-01-25 19:47:06,501] {subprocess.py:85} INFO - Output:
[2022-01-25 19:47:06,505] {subprocess.py:89} INFO - -------------------------- EMPIEZA ------------------------
[2022-01-25 19:47:06,506] {subprocess.py:89} INFO - /tmp/***tmpb134doit
[2022-01-25 19:47:06,507] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 19:47:06,556] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=check, execution_date=20220125T194704, start_date=20220125T194706, end_date=20220125T194706
[2022-01-25 19:47:06,607] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 19:47:06,676] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
